friendshipper/src-tauri/src/repo/operations/install_git_hooks.rs

```diff
@@ -64,6 +67,12 @@ impl Task for InstallGitHooksOp {
 
                     debug!("copying {:?} to {:?}", entry.path(), destination);
                     fs::copy(entry.path(), &destination)?;
+
+                    #[cfg(target_os = "macos")]
```

Comment:
wdyt about going for `not(target_os = "windows")` here? so we "accidentally" get linux support